### PR TITLE
Keeps chaplain up to date

### PR DESCRIPTION
**Pull request by Bawhoppen**

### CODE REVIEW

**Comment by GunHog** [comment]
How disappointing. I thought we would be getting more cosmetics for chaplain.

**Comment by Supermichael777** [comment]
not having a God delusion atheism one.

**Comment by TheRealGLH** [comment]
You should also add "cuck", "wew" and "wew lad".

**Comment by bgobandit** [comment]
memes

**Comment by TrustyGun** [comment]
most literal use of the tag ever

File: code/modules/jobs/job_types/civilian_chaplain.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@ Chaplain
 			B.name = "Toolbox Manifesto"
 		if("homosexuality")
 			B.name = "Guys Gone Wild"
-		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks")
+		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
 			B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
```
Comment (Chocobro) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: it should be "Sweet Bro and Hella Jeff: Expanded Edition"
j e f f
not jef
he is not a cleaning product

Comment (Bawhoppen) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: make your own damn PR to fix it

Comment (Chocobro) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: git always crit fails when i'm involved

Comment (PKPenguin321) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: its a one line grammar change, the easiest kind of change
yuo can do it!

**Pull request merged**